### DISCUSSION TITLE
Skip habit this time

### DISCUSSION THREAD

**Discussion by bsaunder2002** [iSoron/uhabits#770] [Feature Requests]
All of my todo's are in Habit.  Brush teeth, eat lunch, check email, exercise, meditate, ...

Sometimes it's necessary to be able to just skip the habit today.  Allow skipping the habit from the reminder notification.

**Comment by iSoron** [comment]
This feature has already been implemented and should become available in Loop 2.0. If you would like to try it out early, we already have [an alpha version available](https://github.com/iSoron/uhabits/releases/tag/v2.0.0-alpha).
